lib/inferno/dsl/fhir_evaluator/data_summary.rb

```diff
@@ -0,0 +1,97 @@
+module FhirEvaluator
+  # DataSummary represents the results of performing data characterization.
+  class DataSummary
```

Comment:
Unless DataSummary is actually used anywhere, let's leave it out until we need to bring it back in

Reply:
Done https://github.com/inferno-framework/inferno-core/pull/557/commits/be8db9fc65e60a9af26649991d8bc904337ad676